website/docs/api/admin/events-api.md

```diff
@@ -1557,3 +1557,24 @@ This event fires when you delete a segment.
   "environment": null
 }
 ```
+
+## Suggest changes events
+
+### 'suggest-change-created'
+
+This event fires when you create a a suggest-changes draft.
+
+```json title="example event: suggest-change-created"
+{
+  "id": 971,
+  "type": "suggest-change-created",
+  "createdBy": "user@company.com",
+  "createdAt": "2022-06-03T10:30:08.128Z",
+  "data": {},
+  "preData": null,
+  "tags": [],
+  "featureName": null,
+  "project": null,
+  "environment": null
+}
+```
```

Comment:
Nice! Thank you! Is this what the actual event will look like? Does it contain any data or anything? I'd expect there to be a link to the suggestion that was created, at least? This example should show more or less exactly what the users would see when creating a change suggestion.

Reply:
Would you add the id of the change request or the whole request?

Reply:
Whatever we want to put in the event. How have you designed the event? Having both would provide the maximum flexibility, but I don't know how large these events would be. At the very least, you should be able to go and see the resource that was created, right?